lib/animina_web/live/flags_live.ex

```diff
@@ -34,7 +34,7 @@ defmodule AniminaWeb.FlagsLive do
     socket
     |> assign(page_title: gettext("Select your own flags"))
     |> assign(color: :white)
-    |> assign(navigate_to: "/profile/flags/green")
+    |> assign(navigate_to: "/profile/green-flags")
```

Comment:
`/profile/flags/green` is correct. That shouldn't be undone with this PR. Same for the other flag routes in this PR.

Reply:
I've reverted this